packaging/run_unittests_binaries.sh

```diff
@@ -13,24 +13,29 @@ export GST_PLUGIN_PATH=$(pwd)/gst/nnstreamer
 export NNSTREAMER_CONF=$(pwd)/nnstreamer-test.ini
 export NNSTREAMER_FILTERS=$(pwd)/ext/nnstreamer/tensor_filter
 export NNSTREAMER_DECODERS=$(pwd)/ext/nnstreamer/tensor_decoder
-export PYTHONPATH=$(pwd)/ext/nnstreamer/tensor_filter/:$PYTHONPATH
-
+export _PYTHONPATH=${PYTHONPATH}
 
 run_entry() {
   entry=$1
-
-  if [[ $entry == *"python3"* ]]; then
+  if [[ $entry == *"python3"* || $entry == *"python2"* ]]; then
+    PY=$(echo ${entry} | grep -oP "python[0-9]")
     pushd ext/nnstreamer/tensor_filter
-    ln -sf nnstreamer_python3.so nnstreamer_python.so
+    TEST_PYTHONPATH=${PY}_module
+    rm -rf ${TEST_PYTHONPATH}
+    mkdir -p ${TEST_PYTHONPATH}
+    pushd ${TEST_PYTHONPATH}
+    # Covert to an absolute path from the relative path
+    TEST_PYTHONPATH=$(pwd)
+    export PYTHONPATH=${TEST_PYTHONPATH}
```

Comment:
Should this be `PYTHONPATH=${TEST_PYTHONPATH}:${_PYTHONPATH}` ?

`meson.build` declares requirement for `numpy` when using python. This might override it.

Reply:
@zhoonit nope. PYTHONPAYH is an extra path. Numpy will be imported from the system path.